.github/actions/check_compliance/action.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Check External Contribution'
```

Comment:
The name and description is the same for both actions. I think this one should be named differently - to reflect that it is compliance check.